agents/otlp/src/otlp_metrics.cc

```diff
@@ -54,6 +56,23 @@ static std::vector<std::string> discarded_metrics = {
   "thread_id", "timestamp"
 };
 
+OTLPMetrics::OTLPMetrics(uv_loop_t* loop, const OTLPAgent& agent):
+    start_(duration_cast<time_point::duration>(
+              microseconds(static_cast<uint64_t>(
+                  performance::performance_process_start_timestamp)))),
+    agent_(agent) {
+  const std::string prot = GetOtlpDefaultHttpMetricsProtocol();
+  fprintf(stderr, "OTLPMetrics::OTLPMetrics: prot: %s\n", prot.c_str());
```

Comment:
Should this be placed `Debug()` instead?

Reply:
I forgot to remove this. Thanks for pointing this out